### ISSUE TITLE
Broken test for protocol versions

### ISSUE THREAD

**Issue by platfowner** [ainblockchain/ain-blockchain#131]
npm run test_blockchain
```
  ...
    Protocol versions
      ✓ accepts API calls with correct protoVer
(node:73098) UnhandledPromiseRejectionWarning: AssertionError: expected undefined to equal 1
    at Promise.all.then.res (/Users/seo/git/comcomai/blockchain/ain-blockchain/integration/blockchain.test.js:679:34)
    at process._tickCallback (internal/process/next_tick.js:68:7)
(node:73098) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:73098) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.


      1) rejects API calls with incorrect protoVer
      ✓ rejects API calls with no protoVer
  ...
```

**Comment by platfowner** [comment]
@liayoo I'm not quite sure which commit it's from. Could you take a look? I ran the tests on the 'develop' branch, fyi.

**Comment by liayoo** [comment]
Thanks for the report. I'm on it!

**Comment by platfowner** [comment]
Fixed by #132.
